modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/view_tree.jsp

```diff
@@ -132,7 +132,6 @@ if (organization != null) {
 			>
 				<liferay-ui:search-container-row
 					className="Object"
-					escapedModel="<%= true %>"
```

Comment:
@stsquared99 @drewbrokke
FYI: This causes a double escape when `displayStyle` is `icon`